Detectors/TPC/reconstruction/include/TPCReconstruction/TrackTPC.h

```diff
@@ -0,0 +1,108 @@
+/// \file TrackTPC.h
+/// \brief Definition of the TPC Track
+/// \author Thomas Klemenz, TU Muenchen, thomas.klemenz@tum.de
+
+
+#ifndef ALICEO2_TPC_TRACKTPC
+#define ALICEO2_TPC_TRACKTPC
+
+#include "DetectorsBase/Track.h"
+
+#include "TPCSimulation/Cluster.h"
+
+#include <TClonesArray.h>
+
+namespace o2 {
+namespace TPC {
+
+/// \class TrackTPC
+/// This is the definition of the TPC Track Object
+
+
+class TrackTPC final : public o2::Base::Track::TrackParBase {
+  public:
+
```

Comment:
@wiechula : bit general comment: the TrackParBase class was meant to be used only as a base class for final classes TrackParCov (parametrization with error matrix) and TrackPar (parametrization w/o error matrix). I was thinking to have these TrackParCov and TrackPar as data members of more complex track classes rather than the top base class. In this particular case it is probably ok to use TrackParBase as you did, for the beam test only, since you anyway store the track w/o covariance. But for the general output TPC track class let's discuss separately which layout will be the best one.